presto-hive/src/main/java/com/facebook/presto/hive/util/SerDeUtils.java

```diff
@@ -298,9 +268,46 @@ private static Timestamp getTimestamp(Object object, TimestampObjectInspector in
         return inspector.getPrimitiveJavaObject(object);
     }
 
-    public enum JsonContext
+    private static Type getSerializerType(ObjectInspector inspector)
```

Comment:
I was able to get `Type` parameter from callers, as they all already have `types` variable